packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -454,6 +455,15 @@ class RouteSettings {
   /// If null, the route is anonymous.
   final String name;
 
+  /// Whether this route is the very first route being pushed onto this [Navigator].
+  ///
+  /// The initial route typically skips any entrance transition to speed startup.
```

Comment:
The doc comment here should mention that this is deprecated and point to the alternative as well.